Dockerfile

```diff
@@ -16,7 +16,7 @@ ENV NODE_EXTRA_CA_CERTS=/usr/local/share/ca-certificates/internal-ca.crt
 
 # We need a basic init process to handle signals and reap zombie processes, tini handles that
 # Install Internal CA certificate
-RUN apk update && apk add --no-cache tini && apk add ca-certificates && rm -rf /var/cache/apk/*
+RUN apk update && apk add --no-cache tini 'libretls=~3.3.4-r3' && apk add ca-certificates && rm -rf /var/cache/apk/*
```

Comment:
This had to be a specific patch release as greater than 3.3.4 causes conflicts with an existing library.